config/core/roles/webhook-clusterrole.yaml

```diff
@@ -98,6 +98,18 @@ rules:
       - "leases"
     verbs: *everything
 
+  # For creating events
+  - apiGroups:
```

Comment:
can you run the `make RELEASE.....` target on the make file?

this updates the midstream manifests

Reply:
the full command would be:
```
make generate-release RELEASE=v1.8
```

Reply:
No, it's that we have inconsistency and every repo accepts it's own "RELEASE" format

Reply:
No, it is supposed to be `make RELEASE=v1.8`, the fact that you need to do `knative-v1.8.0` is IMO wrong.

Our "older" branches work just with `v1.6`, for instance:
https://github.com/openshift-knative/eventing/blob/release-v1.6/openshift/release/artifacts/eventing-core.yaml#L20
(and the matching SO repo: https://github.com/openshift-knative/serverless-operator/blob/main/openshift-knative-operator/cmd/operator/kodata/knative-eventing/1.6/2-eventing-core.yaml#L20)

But for 1.8, this is different, on branch creation

Reply:
Yeah, the release param here: https://github.com/openshift-knative/eventing/blob/8e00ab32a7fcc02324590cfb6e2fb063786db137/openshift/release/generate-release.sh#L21

is also expecting only v1.8, so I'm not sure why it worked with the knative-v1.8.0